### DISCUSSION TITLE
Setting for not integrating in the page-model

### DISCUSSION THREAD

**Discussion by rokdd** [wagtail/wagtail-localize#404] [Q&A] [answered]
I am new to wagtail and all this stuff, but I decided that I do not want the page nested as tree to translate them and choose for this part the `wagtail-modeltranslation`. But to translate the templates I need to use wagtail-localize and so my question: Is there a option to not integrate into the page-models?

**Comment by kaedroho** [comment, marked as answer]
Hi @rokondo, Wagtail Localize doesn't help with translating templates, you should use [Django for that](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#internationalization-in-template-code).

You can't use ``wagtail-modeltranslation`` and Wagtail's built-in internationalisation features together, so I'd recommend uninstalling ``wagtail-localize`` and disabling Wagtail's built-in internationalisation.

**Reply by rokdd** [reply]
Okay thank you for clarifying this. Is there a instruction how to uninstall wagtail-localize clean? Because I just uninstalled with pip but I have still the synced tree with the aliases and the field definitions in the model? Thank you!

**Reply by kaedroho** [reply]
Hi @rokdd 
Alias pages are a Wagtail core feature, so you will have to remove these manually. No new alias pages will be created after uninstalling Wagtail Localize though.

Thanks,
Karl
